tools/devtools/templates/new-in-devtools-image-snippet.md

```diff
@@ -0,0 +1,25 @@
+<!-- 1. Save the script below as DevTools snippet. See https://developer.chrome.com/docs/devtools/javascript/sources/#snippets. -->
+<!-- 2. Upload the images in _temp folder manually -->
```

Comment:
I was just trying this out and wondered where to upload them?

Reply:
https://web-dev-uploads.web.app/uploader

Reply:
It would be nice if I can just upload it using firebase api and get the url. But I don't have capacity to dig into that.